public-api/src/test/resources/application.properties

```diff
@@ -2,5 +2,5 @@ spring.datasource.driver-class-name=org.h2.Driver
 spring.datasource.url=jdbc:h2:mem:db;DB_CLOSE_DELAY=-1;INIT=CREATE SCHEMA IF NOT EXISTS workbench
 spring.datasource.username=sa
 spring.datasource.password=sa
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
what does this change do ? What is this for?

Reply:
this tells spring to apply new schema changes if they're not in the db already. Taking away the `drop` means that spring will no longer drop tables/columns that are not referenced in the code.

Reply:
Peter, Greg
I seem to get an error saying table achilles_results not found after unsuccessful alter statement and tests failed.
According to this:
https://stackoverflow.com/questions/12054422/unsuccessful-alter-table-xxx-drop-constraint-yyy-in-hibernate-jpa-hsqldb-standa
hibernate seems to attempt to drop the constraints/data or whatever even from the new database and that seems to cause the issue.
Changing this to just create helped the tests to run and pass.